ui/src/components/PortfolioLog.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import { useAppStore } from '../state';
+
+type PortfolioEvent = {
+  timestamp: string,
+  retainerBalance: bigint,
```

Comment:
should be an `Amount` with a `Brand` (or... hm... maybe an `AmountArg` that can have a `Brand` or a `Denom`)